src/components/custom/_eu-redirect-modal.js

```diff
@@ -79,6 +82,12 @@ const Cancel = styled.span`
         background-color: rgba(0, 0, 0, 0.08);
     }
 `
+
+const StyledText = styled(Text)`
+    @media (max-width: 380px) {
+        font-size: 7px;
```

Comment:
i think the font size is too small, the minimum text we had is 10px

Reply:
I guess i just remove the media query for this styled text